Ficek_QuantiApp/Ficek_QuantiApp/Domain/Sources/RocketClient/ModelConverter/DepencendyValues+rocketConverter.swift

```diff
@@ -0,0 +1,59 @@
+import Foundation
```

Comment:
Pojmenuj ten adresar prosim `Converters`. Mas tam treba `Models`, taky to nemas jednotnym cislem. Navic to slovo `Model` je tam zbytecny, to je jasny ze `converters` delaji konverze modelu. Premejslej vic nad tou konzistenci.